app/models/order.rb

```diff
@@ -267,6 +268,28 @@ def can_transition
     return (required_process_checks - process_checklists).empty?
   end
 
+  def send_submission_pickup_email?
+    booking_type.eql?(BookingType.appointment) || order_transport&.pickup?
```

Comment:
Could you use `booking_type.appointment?` instead ?

There is a (very very very unlikely) potential edge case. Since booking types are defined by a string (e.g "appointment), it technically opens up the possibility of having multiple booking types with the same identifier (for example if the seed task is faulty and creates duplicates).

So it's better to verify the identifier (that's what `appointment?` does), as opposed to comparing to another record.

This is really

Reply:
yeah, it makes sense. Please re-check, I've changed it.

Reply:
looks good !